source/franka_matlab/nvidia_jetson_hardware_support.rst

```diff
@@ -0,0 +1,67 @@
+NVIDIA Jetson Hardware Support
+==============================
+
+The Simulink implementations with the Franka MATLAB Toolbox can be readily built and deployed 
+for NVIDIA's Jetson Hardware boards, by both Linux and Windows host PCs.
+
+Installation and Requirements
+-----------------------------
+
+Please make sure that you are using the `Jetson Linux Real-Time Kernel Package <https://docs.nvidia.com/jetson/archives/r35.1/DeveloperGuide/text/SD/Kernel/KernelCustomization.html#using-the-jetson-linux-real-time-kernel-package>`_.
```

Comment:
Could you clarify in this sentence that in case of not having the ai companion, then people need to make sure that they use the Jetson Linux Real-Time Kernel Package?

Reply:
fixed